### PR TITLE
User may view an item's unit of measures information

### PR DESCRIPTION
**Pull request by LucasAnderson1984**
Update item front end so that a user may view its unit of measures.

needed-by #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,25 @@ Item Manufacturing and Sales
 
 # Purpose
 This is a project for CSU Fresno CSCI 226. The concept is to show how relational databases work. The project is going to use a real world example of how items are built from raw materials and then selling those items to customers.
+
+# Trigger
```
Comment (josephbridgwaterrowe) [inline README.md]: Why would you do this?

Comment (LucasAnderson1984) [reply README.md]: The professor is requiring a trigger so I thought this was the easiest and something I would want done anyways.

Comment (josephbridgwaterrowe) [reply README.md]: Bad professor.

Comment (josephbridgwaterrowe) [reply README.md]: If you must include it I would at least make sure it is up/down'd by a migration.

File: db/migrate/20191203025025_create_item_unit_of_measure_trigger.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+class CreateItemUnitOfMeasureTrigger < ActiveRecord::Migration[6.0]
+  def up
+    ActiveRecord::Base.connection.execute(
+      'create or replace function insert_item_unit_of_measure() ' \
```
Comment (josephbridgwaterrowe) [inline db/migrate/20191203025025_create_item_unit_of_measure_trigger.rb]: It would probably ready easier if the SQL keywords were upper case :smile:

**Pull request merged**